### PR TITLE
add parameters 23, 29

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: docs/parameter-types.md [hunk 1]
```diff
@@ -225,6 +225,51 @@ It is a `3`-byte unsigned int BE, real frequency value divided by `100`.
 Message hex dump with LRC: `03 05 12 05 00 00 c8 8c`
 
 
+## Absolute data
+
+Parameter of that type used to setup absolute data for not multichannel device
```
Comment (DarkPark) [inline docs/parameter-types.md:230]:
```suggestion
Parameter is used to setup absolute data for not multichannel device.
```

File: docs/parameter-types.md [hunk 2]
```diff
@@ -225,6 +225,51 @@ It is a `3`-byte unsigned int BE, real frequency value divided by `100`.
...
+
+#### **meter value**
+
+Base absolute meter value.
+<br>
+It is a `4`-byte unsigned int BE.
+
+#### **pulse coefficient**
+
+[Pulse coefficient](./types.md#pulse-coefficient)
```
Comment (DarkPark) [inline docs/parameter-types.md:249]:
```suggestion
[See details](./types.md#pulse-coefficient).
```

?

File: docs/parameter-types.md [hunk 3]
```diff
@@ -257,3 +302,56 @@ In `OTAA` connection mode, after more than `5` unsuccessful attempts to connect
 | interval       | `3600` | `0x100e` |
 
 Message hex dump with LRC: `03 03 1c 10 0e 57`
+
+
+## Absolute data multi channel
+
+Parameter of that type used to setup absolute data for multichannel device
```
Comment (DarkPark) [inline docs/parameter-types.md:309]:
```suggestion
Parameter is used to setup absolute data for multichannel device.
```

File: docs/parameter-types.md [hunk 4]
```diff
@@ -257,3 +302,56 @@ In `OTAA` connection mode, after more than `5` unsuccessful attempts to connect
...
+
+#### **meter value**
+
+Base absolute meter value.
+<br>
+It is a `4`-byte unsigned int BE.
+
+#### **pulse coefficient**
+
+[Pulse coefficient](./types.md#pulse-coefficient)
```
Comment (DarkPark) [inline docs/parameter-types.md:335]: as above

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: docs/parameter-types.md (hunk 1 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed in all descriptions

File: docs/parameter-types.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

File: docs/parameter-types.md [hunk 5]
```diff
@@ -257,3 +302,56 @@ In `OTAA` connection mode, after more than `5` unsuccessful attempts to connect
...
+
+| Size | Type   | Field                                          |
+| ---- | ------ | ---------------------------------------------- |
+| 1    | `byte` | parameter type = `29`                          |
+| 1    | `byte` | [channel index](#channel-index)                |
+| 1    | `byte` | [meter value](#meter-value)                    |
+| 3    | `byte` | [pulse coefficient](#pulse-coefficient)        |
+| 3    | `byte` | [pulse counter's value](#pulse-counters-value) |
+
+#### **channel-index**
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

File: docs/parameter-types.md (hunk 4 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

File: docs/parameter-types.md [hunk 6]
```diff
@@ -257,3 +302,56 @@ In `OTAA` connection mode, after more than `5` unsuccessful attempts to connect
...
+
+#### **pulse counter's value**
+
+Base pulse counter's value.
+<br>
+It is a `4`-byte unsigned int BE.
+
+### Examples
+
+#### set absolute data (not multichannel device):
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**